### DISCUSSION TITLE
Lighthouse (Bonnes pratiques)

### DISCUSSION THREAD

**Discussion by ThomasErhel** [cnumr/EcoIndex#219] [Bug]
Bonjour! 👋

⚠️ **Les liens vers les destinations multi-domaines sont dangereux**

Ajoutez les attributs `rel="noopener"` ou `rel="noreferrer"` à tous les liens externes pour améliorer les performances et prévenir les failles de sécurité. [En savoir plus](https://developer.chrome.com/docs/lighthouse/best-practices/external-anchors-use-rel-noopener/)

![Lighthouse](https://user-images.githubusercontent.com/16942870/209149075-8f5e3674-de56-4527-9d78-81a4120a92e1.png)

**Comment by yaaax** [comment]
Issue créée : #220
